### DISCUSSION TITLE
imtoken钱包的deapp里面， 波卡消息签名调不起钱包的签名

### DISCUSSION THREAD

**Discussion by mahuanhuan** [consenlabs/webview#26] [General]
https://polkadot.js.org/docs/extension/cookbook
![image](https://user-images.githubusercontent.com/15261282/133252904-5d70eb9c-b8fe-49ef-8df6-574d252cf262.png)

**Comment by unix** [comment]
Related issues: #24

**Comment by xwartz** [comment]
Sorry, the `signRaw` method is not implemented on imToken, we only support the `signPayload` method for now.
